packages/mui-joy/src/Link/Link.tsx

```diff
@@ -216,26 +216,18 @@ const Link = React.forwardRef(function Link(inProps, ref) {
 
   const level = nesting || inheriting ? inProps.level || 'inherit' : levelProp;
 
-  const {
-    isFocusVisibleRef,
-    onBlur: handleBlurVisible,
-    onFocus: handleFocusVisible,
-    ref: focusVisibleRef,
-  } = useIsFocusVisible();
   const [focusVisible, setFocusVisible] = React.useState<boolean>(false);
-  const handleRef = useForkRef(ref, focusVisibleRef) as React.Ref<HTMLAnchorElement>;
+  const handleRef = useForkRef(ref) as React.Ref<HTMLAnchorElement>;
```

Comment:
Is this `useForkRef` needed? Given it only receives `ref`.

Reply:
Removed 👍🏼